Tasks/XamarinAndroidV1/task.json

```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 229,
+        "Minor": 230,
         "Patch": 0
```

Comment:
[not blocking]

We shouldn't be required to bump the base task version if there are no pre-compilation instructions; I'm curious there was a check that required prompted the bump.

If we do bump the base version, please check for overrides (and bring them forward to the new version)

Reply:
Sure we'll check the overrides for them. There's a before build check for these base task versions that needs to be in current sprint x.230.x for all base tasks.

Reply:
>There's a before build check for these base task versions that needs to be in current sprint x.230.x for all base tasks.

Assuming the base task didn't change, that should occur in two situations:
1.  The base task is under _generated\BaseTask
2. Preprocessor is used (thus _generated\BaseTask is required)

If the two conditions are not met, base task version bump should not be required.  Please raise if that is not the case, so we can check into it